src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -621,6 +621,10 @@ export class QuickBar extends LitElement {
           --ha-chip-text-color: black;
         }
 
+        span.command-text {
+          margin-left: 6px;
```

Comment:
It should be either 4 or 8 pixels

Reply:
I went with 6 since that is the used `border-radius` of the chip. 4 and 8 both look slightly "off" since not symmetric with the space inside the chip.

If you still want me to change it, I would go with 4 since 8 is too much.

Reply:
All sizes should be a multiple of 4, the border radius is 16px for example.

I would say it should have at least 8px between the chip and the text:

![image](https://user-images.githubusercontent.com/5662298/112667315-6a36a180-8e5d-11eb-86ad-2d8847de8072.png)

Reply:
Mhh...I saw this `border-radius` here, but apparently that takes no effect or I am missing where it does? https://github.com/home-assistant/frontend/blob/6de8b4e35ff904b1f755c8923e905451100d4be0/src/dialogs/quick-bar/ha-quick-bar.ts#L601-L607

Either way, I changed it now to `8px`.

Reply:
Right... Those styles are not correct 😅